src/gui/advancedsettings.cpp

```diff
@@ -499,6 +539,15 @@ void AdvancedSettings::loadAdvancedSettings()
     comboBoxSeedChokingAlgorithm.addItems({tr("Round-robin"), tr("Fastest upload"), tr("Anti-leech")});
     comboBoxSeedChokingAlgorithm.setCurrentIndex(static_cast<int>(session->seedChokingAlgorithm()));
     addRow(SEED_CHOKING_ALGORITHM, tr("Upload choking algorithm"), &comboBoxSeedChokingAlgorithm);
+    item(SEED_CHOKING_ALGORITHM, PROPERTY)->setToolTip(QString("<body><p>%1</p></body>")
+            .arg(tr("Controls the seeding unchoke behavior. The available options are:<br><br>"
+                    "<strong>Round-robin</strong>: which round-robins the peers that are unchoked when seeding. "
+                    "This distributes the upload bandwidht uniformly and fairly. It minimizes the ability"
+                    " for a peer to download everything without redistributing it<br><br>"
```

Comment:
@thalieht:  Some poor phrasing, and a typo. 

>which **rotates** the peers 

>the upload **bandwidth** uniformly 

"Round-robin" is a *noun*, not a *verb*.  "Rotate" is the closest *verb* to "round-robin", in meaning.

Reply:
Fixed.